map/framework.cpp

```diff
@@ -2448,21 +2448,24 @@ bool Framework::ParseEditorDebugCommand(search::SearchParams const & params)
 
 namespace
 {
-vector<string> FilterNearbyStreets(vector<search::ReverseGeocoder::Street> const & streets,
-                                   string const & exactFeatureStreet = "")
+vector<osm::LocalizedStreet> FilterNearbyStreets(vector<search::ReverseGeocoder::Street> const & streets,
+                                                 string const & exactFeatureStreet = "")
 {
-  vector<string> results;
+  vector<osm::LocalizedStreet> results;
   // Exact feature street always goes first in Editor UI street list.
+
+  // TODO: Push into result LocalizedStreet object with default and localized street name.
   if (!exactFeatureStreet.empty())
-    results.push_back(exactFeatureStreet);
+    results.push_back({exactFeatureStreet, ""});
   // Reasonable number of different nearby street names to display in UI.
   constexpr size_t kMaxNumberOfNearbyStreetsToDisplay = 8;
   for (auto const & street : streets)
   {
     auto const e = results.end();
-    if (e == find(results.begin(), e, street.m_name))
+    osm::LocalizedStreet const st {street.m_name, ""};
+    if (e == find(results.begin(), e, st))
```

Comment:
Iterators may be invaildated in various ways and I guess this is not the thing you'd like to debug in the friday's night.